src/components/Blog/CustomRichBlocks.style.ts

```diff
@@ -0,0 +1,72 @@
+import { Typography, alpha, styled } from '@mui/material';
+import { urbanist } from 'src/fonts/fonts';
+
+// Paragraph
+
+export const BlogParagraph = styled(Typography)(({ theme }) => ({
+  color: alpha(
+    theme.palette.mode === 'light'
+      ? theme.palette.black.main
+      : theme.palette.white.main,
+    0.75,
+  ),
+  margin: theme.spacing(2, 0),
+  fontSize: '18px',
+  lineHeight: '32px',
+  fontWeight: 400,
+}));
+
+// Headings
+export const BlogH1 = styled(Typography)(({ theme }) => ({
```

Comment:
Seems a lot of style override, can't we apply that at the theme level? there's also a lot of duplication between each H* variants

Reply:
Good point! Moved it to the theme-level